core/modules/file/file.entity.inc

```diff
@@ -126,6 +128,58 @@ class File extends Entity {
       'options' => array(),
     );
   }
+
+  /**
+   * Overrides Entity::createAccess().
+   */
+  public static function createAccess($bundle = NULL, $account = NULL) {
+    // If no user object is supplied, the access check is for the current user.
+    if (empty($account)) {
+      $account = $GLOBALS['user'];
+    }
+
+    return user_access('access content', $account);
+  }
+
+  /**
+   * Overrides Entity::access().
+   *
+   * @param string $op
+   *   The operation to be performed on the file. Possible values are:
+   *   - "create".
+   *   - "update".
+   *   - "delete".
```

Comment:
Seems like "view" could theoretically be added for File objects through `file_download_access()`, but I would be fine adding that as a follow-up.

Reply:
Let's do that as a follow-up/